techmatters/flex-plugins
# CHI-2191: Add to New Case

## Description

- Ability to add existing contacts to a new case.

### Checklist

### Other Related Issues

None

[CHI-2191](https://tech-matters.atlassian.net/browse/CHI-2191)

### Verification steps


Search for a contact
Select a contact - there should be a blue banner if the contact has been added to a case or the "Add to case" button if vice versa.
Click on the "Add to case" button and select the "New case" button
Verify that the contact was added to a newly created case

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

[CHI-2191]: https://tech-matters.atlassian.net/browse/CHI-2191?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
